packages/wallet-sdk/src/CoinbaseWalletSDK.ts

```diff
@@ -13,11 +13,12 @@ import { MobileRelay } from './relay/MobileRelay';
 import { WalletLinkRelay } from './relay/WalletLinkRelay';
 import { WalletSDKRelayEventManager } from './relay/WalletSDKRelayEventManager';
 import { getFavicon, isMobileWeb } from './util';
+import { LIB_VERSION } from './version';
 
-const LINK_API_URL = process.env.LINK_API_URL || 'https://www.walletlink.org';
+const LINK_API_URL =
+  (typeof process !== 'undefined' && process.env?.LINK_API_URL) || 'https://www.walletlink.org';
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const SDK_VERSION = process.env.SDK_VERSION! || require('../package.json').version || 'unknown';
+const SDK_VERSION = (typeof process !== 'undefined' && process.env?.SDK_VERSION) || LIB_VERSION;
```

Comment:
We should just remove the process.env vars.

```suggestion
const SDK_VERSION = LIB_VERSION || require("./package.json").version; // will require a //@ts-expect-error or eslint disable.
```

Reply:
:+1:

Reply:
I replaced SDK_VERSION with LIB_VERSION since it's always defined in version.ts so require("./package.json") will never run.